include/d/actor/d_a_dk.h

```diff
@@ -1,23 +1,65 @@
 #ifndef D_A_DK_H
 #define D_A_DK_H
 
+#include "d/d_bg_s_acch.h"
+#include "d/d_particle.h"
 #include "f_op/f_op_actor.h"
+#include "m_Do/m_Do_hostIO.h"
 
-struct tail_s {
-    
-};
+class tail_s {
+    public:
```

Comment:
I think names ending in `_s` and `_struct` are intended to be structs, while ones ending in `_c` and `_class` are intended to be classes (though it doesn't make a practical difference besides default visibility)
```suggestion
struct tail_s {
```

Reply:
Aah that makes sense, changed.